UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -46,6 +46,17 @@ public class UnityMeshData
 		public BoneWeight[] BoneWeights;
 	}
 
+	public class ImportProgress
+	{
+		public bool IsDownloaded = false;
+
+		public int NodeTotal = 0;
+		public int NodeLoaded = 0;
+
+		public int TextureTotal = 0;
+		public int TextureLoaded = 0;
+	}
```

Comment:
Consider adding a public float Progress with a getter that looks at the loaded vs totals and returns a progress percentage.  It would be a helper function for callers.  I assume you'd need to write that logic to drive a progress bar somewhere anyway.  Maybe it could be a nullable float where it's null (presented to user as indeterminate progress) until isDownloaded is true?

Reply:
Great idea.  I'll just have it return 0% for now since that will be basically the same.  For now, I'm saying that the progress is Loaded/Total including nodes AND textures.  Not everything takes the same amount of time, but hopefully this will be reasonable.

Reply:
Sounds good.  We can always improve the accuracy later if needed but this is a great improvement and is probably good enough.